apps/meteor/client/startup/e2e.ts

```diff
@@ -146,11 +147,33 @@ Meteor.startup(() => {
 				return message;
 			}
 
+			const mentionsEnabled = settings.get<boolean>('E2E_Enabled_Mentions');
+
+			if (mentionsEnabled) {
+				const me = (message.u?._id && Users.findOne(message.u?._id, { fields: { username: 1 } })?.username) || '';
```

Comment:
Feel this could be a function and do 
```
message.e2eMentions = getE2EMentions()

Reply:
It could, but I don't see the benefit in this case... this is going to be used only in that place

Reply:
Is maybe not to reuse the code but to not to make the nested part too big.